ui/tasklist/client/scripts/process/plugins/action/modals/cam-tasklist-process-start-modal.js

```diff
@@ -79,7 +79,7 @@ module.exports = [
     processStartData.set('currentProcessDefinitionId', {id: null});
 
     var DEFAULT_OPTIONS = ($scope.options = {
-      hideCompleteButton: true,
+      hideCompleteButton: false,
```

Comment:
Can we rename this option to `hideStartButton`? Since it seems like the feature never worked correctly it shouldn't be a problem, right?